src/components/app/details/appDetails/AppDetails.tsx

```diff
@@ -433,7 +444,11 @@ export const Details: React.FC<DetailsType> = ({
         toggleDetailedStatus(false)
     }
 
-    if (!loadingResourceTree && (!appDetails?.resourceTree || appDetails?.resourceTree?.nodes?.length <= 0) && !isVirtualEnvironment) {
+    if (
+        !loadingResourceTree &&
+        (!appDetails?.resourceTree || appDetails?.resourceTree?.nodes?.length <= 0) &&
```

Comment:
we can write this (!appDetails?.resourceTree || !appDetails.resourceTree.nodes?.length) &&